From ef62408248f975dab68f99e1be1eb9836374dc7c Mon Sep 17 00:00:00 2001
From: Jakub Filak <jfilak@redhat.com>
Date: Fri, 27 Mar 2015 10:51:52 +0100
Subject: [PATCH] vmcore: generate 'reason' file in all cases

If kdump generates the dmesg log file (vmcore-dmesg.log), the vmcore's
post-create event doesn't generate 'reason' file. It is caused by
inappropriate use of 'abrt-dump-oops' where the event uses that helper
to parse the log file to generate 'backtrace' file instead of using it
with the '-u' argument to update the dump directory and create all
necessary files.

Resolves: rhbz#1250337

Signed-off-by: Jakub Filak <jfilak@redhat.com>
---
 src/plugins/vmcore_event.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/plugins/vmcore_event.conf b/src/plugins/vmcore_event.conf
index 34608d9..5957b3f 100644
--- a/src/plugins/vmcore_event.conf
+++ b/src/plugins/vmcore_event.conf
@@ -3,7 +3,7 @@ EVENT=post-create analyzer=vmcore
         # If kdump machinery already extracted dmesg...
         if test -f vmcore-dmesg.txt; then
             # ...use that
-            abrt-dump-oops -o vmcore-dmesg.txt >backtrace || exit $?
+            abrt-dump-oops -u $DUMP_DIR vmcore-dmesg.txt || exit $?
             #
             # Does "kernel" element exist?
             test -f kernel && exit 0
-- 
1.8.3.1

